src/auxkernels/Current.C

```diff
@@ -25,6 +25,7 @@ CurrentTempl<is_ad>::validParams()
 
   params.addRequiredCoupledVar("density_log", "The electron density");
   params.addRequiredCoupledVar("potential", "The potential");
+  params.addParam<int>("component", 0, "The component of position. (0 = x, 1 = y, 2 = z)");
```

Comment:
I think the wording for this parameter description is not as intuitive as it could be. Perhaps this should be something like the component of the current vector and this change should be made in the other files as well